### PR TITLE
Feat/filters

### PR DESCRIPTION
**Pull request by adipaul1981**
image

### CODE REVIEW

File: packages/style/themes/default/_colors.scss [hunk 1]
```diff
@@ -1 +1,11 @@
...
\ No newline at end of file
+$white: #fff;
+
+//Move to less in specific themes
+$blue-7: #008FC7;
+
+$gray-3: #F0F3F8;
+$gray-4: #D8E1EB;
+$gray-5: #B5C6D8;
+$gray-6: #90A9C1;
```
Comment (kjacqueschu) [inline packages/style/themes/default/_colors.scss:9]: remove

File: packages/ui/src/components/filters/FilterContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+import {IFilter, IFilterGroup, onChangeType, VisualType} from './types';
+import StackLayout from "../../layout/StackLayout";
+import FilterSelector from "./FilterSelector";
+
+import styles from '@ferlab/style/components/filters/FilterContainer.module.scss';
+
+const { Panel } = Collapse;
+
+type FilterContainerProps = {
+    filterGroup: IFilterGroup;
```
Comment (kjacqueschu) [inline packages/ui/src/components/filters/FilterContainer.tsx:14]: why did we remove the dictionary type ? I think it was useful for translations

**Review by kjacqueschu — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/filters/FilterContainer.tsx (hunk 2 above)
Comment (kjacqueschu) [reply packages/ui/src/components/filters/FilterContainer.tsx:14]: now its impossible to translate

Comment (adipaul1981) [reply packages/ui/src/components/filters/FilterContainer.tsx:14]: in KF we do not have translations. we dont need a dictionary. I think the translation should be done in cqdg in the input should be the result of that.

**Pull request merged**